MapboxNavigation/InstructionsCardViewController.swift

```diff
@@ -13,7 +13,7 @@ open class InstructionsCardViewController: UIViewController {
     var instructionsCardLayout: InstructionsCardCollectionLayout!
     
     public private(set) var isInPreview = false
-    private var currentStepIndex: Int?
+    public var currentStepIndex: Int?
```

Comment:
How do we expect the application to use this property?

Reply:
@d-prukop - Can you comment on what you use this property for in Apex? Is it possible we can avoid needing it in a client app?

Reply:
@avi-c see my comments below. This and the exposing of `reloadDataSource` are related